how-to/workspace-platform-starter/client/src/framework/workspace/dock.ts

```diff
@@ -445,7 +447,7 @@ async function addEntriesByAppTag(
 			// Individual so show a button for each app
 			for (const dockApp of dockApps) {
 				const icon = entry.iconUrl ?? getAppIcon(dockApp);
-				buttons.push({
+				return {
```

Comment:
This seems like a breaking change. Right now you can specify a tag and say individual (i.e. spread them on the root) and it will add a button to the dock for each app that gets matched. This seems to just return the first one rather than add a list of apps across the dock.

Reply:
ah yes you're right that's my bad - will fix